### PR TITLE
530 claimant contact confirmation email

### PR DESCRIPTION
**Pull request by cehsu**
Connects to department-of-veterans-affairs/vets.gov-team#3152
Based on feedback to this [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/5752#issuecomment-308832233).

### CODE REVIEW

File: src/schemas/21P-530/schema.js [hunk 1]
```diff
@@ -124,7 +119,16 @@ let schema = {
     },
     amountGovtContribution: {
       type: 'number'
-    }
+    },
+    claimantEmail: {
+      type: 'string',
+      format: 'email'
+    },
+    'view:claimantEmailConfirmation': {
```
Comment (lihanli) [inline src/schemas/21P-530/schema.js]: aren't these fields not supposed to be in the schema?

Comment (cehsu) [inline src/schemas/21P-530/schema.js]: Thanks @lihanli, @J-Quag and @stacyhlinn provided feedback on [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/5752#issuecomment-308832233), indicating that the email should be checked as in EDU. Please let me know if I should approach this differently.

Comment (lihanli) [inline src/schemas/21P-530/schema.js]: i think the view fields are supposed to be used only in on the frontend and removed before sending to the backend, you can add this field after importing the schema and do the email check entirely in the frontend

Comment (cehsu) [inline src/schemas/21P-530/schema.js]: Thanks for the reminder @lihanli, I'll close this.

**Pull request closed without merging**